### DISCUSSION TITLE
Broken Leslie performance question

### DISCUSSION THREAD

**Discussion by plamikcho** [Banana71/Soundplantage#2] [Q&A] [answered]
Hi Peter,

I am curious about one of your latest performances in Greenhouse bank - Broken Leslie. The organ sound is amazing but there is some surrounding noise and something like bells. Is this intentional - as an effect (because the title says "broken")? Or it requires another engine?

**Comment by Banana71** [comment, marked as answer]
Someone hung the wind chime in the Leslie.
"Boken Leslie" is an organ sound with a broken Leslie effect. The crackles and bell noises are caused by an overdriven modulator (OP6).
Algorithm 24; Feedback 7; Operator 6 Output Level 92, Freq Fine between 1 Hz and 2.5 Hz.

I use the same effect in the sound 115_CosmicBreak. The output level is a little smaller and only the crackles occur.


![Windspiel](https://github.com/Banana71/Soundplantage/assets/104296205/21722f00-0888-49d7-8bac-7d2d5584de92)

**Reply by plamikcho** [reply]
Thank you very much 👍
